hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFuture.java

```diff
@@ -171,4 +171,24 @@ private static void appendHeartbeat(StringBuilder sb, long lastHeartbeatMillis)
             sb.append(timeToString(lastHeartbeatMillis)).append(". ");
         }
     }
+
+    @Override
+    public E get() throws InterruptedException, ExecutionException {
+        String name = Thread.currentThread().getName();
+        System.out.println(">>> FATAL " + name);
```

Comment:
is this left over for debugging?

Reply:
I will replace this part, just looking for more efficient solution